src/main/java/jnr/enxio/channels/AbstractNativeServerSocketChannel.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2019 JNR authors
```

Comment:
2019! Also, is “JNR authors” okay?

Reply:
We would prefer actual names although we are not great at updating this.  With no legal entity to assign ownership the term JNR authors I don't think asserts copyright claim (IANAL).

Reply:
Works for me. Fixed.